plugin/evm/atomic_syncer.go

```diff
@@ -107,6 +107,12 @@ func (s *atomicSyncer) onLeafs(keys [][]byte, values [][]byte) error {
 					return err
 				}
 			}
+			// Trie should be re-opened after committing.
+			trie, err := s.atomicTrie.OpenTrie(root)
```

Comment:
Do we only need to do this if `isCommit`?

Reply:
The trie is always committed, `isCommit` determines whether the `trieDB` was committed hence requiring a commit to the versiondb.